### ISSUE TITLE
Transient in NGC galaxy associated with PS1 object in ATLAS but correctly associated in Lasair

### ISSUE THREAD

**Issue by thespacedoctor** [thespacedoctor/sherlock#127]
https://star.pst.qub.ac.uk/sne/atlas4/candidate/1181150931145856800/

**Comment by thespacedoctor** [comment]
Using the coordinates provided from ATLAS gives only one match:

```bash
sherlock -N match -- 18:11:51.08 +14:58:56.0 ~/.config/sherlock/sherlock_atlas4_all.yaml
```

But the coordinates from [Lasair](https://lasair.roe.ac.uk/object/ZTF20aclwclm/) give the multiple matches reported on their pages:

```bash
sherlock -N match -- 18:11:51.098 14:58:56.805 ~/.config/sherlock/sherlock_atlas4_all.yaml
```

The separation between the coordinate sets is 0.9 arcsec (-0.8 N, -0.3 E).

The ATLAS coordinates have pushed the transient location close enough to the PS1 "galaxy" to make it synonymous. In reality the "galaxy' is the core of NGC6574. 

**SOLUTION**: Sherlock currently matches against all synonym search modules and then ONLY IF NO MATCH IS FOUND continues to the association search modules. Now that a) catalogue matches are merged if sherlock thinks they are the same underlying source and b) speed of matching a fat transient stream does not seem to be an issue for ATLAS/PanSTARRS/Lasair, we can now drop this synonym match break and allow the full search algorithm to run on every source.

**Comment by thespacedoctor** [comment]
Here is another example where dropping the synonym match break would allow the underlying NED galaxy to match:
https://star.pst.qub.ac.uk/sne/atlas4/candidate/1232801090020955600/

**Comment by thespacedoctor** [comment]
After extensive tests on the ATLAS volume limited sample, to make sure the quality of match results remain high, I have now switched off the synonym match break by default (will be pushed with next sherlock release). Over all some extra meta-data was added to the volume limited sample results and one of the galaxy hosts was given an alternative name. One source that was incorrectly given a VS designation is now being reported correctly as a SN with the true host association given. So, over all an improvement on matches.

It remains to be seen if this change will negatively affect the number of false-positive SN classifications (true variable-stars labelled as SN). Easy to roll back if we are flooded with VSs pretending to be SNe.
